book/book.tex

```diff
@@ -28,6 +28,7 @@
 
 \usepackage{kantlipsum}
 \usepackage{cleveref}
+\usepackage{hyperref}
```

Comment:
Can you remove this? We eventually need to interact with Springer's typesetting service so minimising the number of new packages is best